src/VisualStudio/Core/Def/ProjectSystem/MetadataReferences/VisualStudioMetadataReferenceManager.cs

```diff
@@ -176,7 +173,7 @@ AssemblyMetadata GetMetadataWorker()
         }
 
         void GetStorageInfoFromTemporaryStorage(
```

Comment:
>GetStorageInfoFromTemporaryStorage [](http://example.com/codeflow?start=13&length=34)

nit: did this capture before too?

Reply:
Yes. It was previously capturing this. I can change this to be static though. Will do when I get in.